docs/examples/testing/large_integration_test.py

```diff
@@ -1,15 +1,12 @@
 """
-In a large integration test of highly connected services, it's preferable to
-limit interactions to only those under test.
-
 This file defines several toy services that interact to form a shop of the
 famous ACME Corporation. The AcmeShopService relies on the StockService,
 InvoiceService and PaymentService to fulfil its orders. They are not best
 practice examples! They're minimal services provided for the test at the
 bottom of the file.
```

Comment:
I'd still follow PEP8 by making adding an `Error` suffix to the exceptions. There's also a confusion between `NotImplementedError` (the built-in exception) and `NotImplemented` (the built-in sentinel for comparators) which isn't a thing that can be raised.